docs/Contribute/ContributeWrite/WriteStack.md

```diff
@@ -0,0 +1,63 @@
+# Contribute to the Software Stack
+
+:::warning
+Writing on the *software stack* makes changes directly to the BTCPay Server software that everyone uses. **Do not** change the code if you aren't familiar with it.
```

Comment:
>Writing on the *software stack* makes changes directly to the BTCPay Server software that everyone uses. 

I'm confused why code changes are in the "WriteStack.md" doc. I think that it should be in software development or coding contribution. WriteStack sounds like documentation. 

>**Do not** change the code if you aren't familiar with it.

I think that we should not make this kind of disclaimer, we want contributions. The PR exists to allow users to make code changes that are reviewed.

Reply:
I second this, no need to put a warring of this sort. Text seems a bit too much, I'd remove this section.

Reply:
@britttttk For context, I occasionally fix typos and reword some paragraphes that can sound confusing or unclear in the `btcpayserver` repo. This is about that kind of contribution.
Not really code for me, since we just touch the wording ?

Will delete the warning.